src/main/java/com/sprint/mission/discodeit/repository/ChannelRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.sprint.mission.discodeit.repository;
+
+import com.sprint.mission.discodeit.entity.Channel;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+public interface ChannelRepository {
+    void createChannel(UUID id, Channel channel);
+    void updateChannelName(UUID id, String name);
+    void updateDescript(UUID id, String descript);
```

Comment:
메서드명이 길어지더라도 Description 모두 적는게 의미가 더 명확할 것 같아요.  

```suggestion
    void updateDescription(UUID id, String description);
```